public/html/reports/ReportsViewModel.html

```diff
@@ -46,7 +46,6 @@
                             <div class="popover-content hidden" data-bind="attr: {'data-id': 'source-'+id}">
                                 <dl class="container col-md-12 dl-horizontal">
                                     <dt>url:</dt><dd data-bind="text: source.url"></dd>
-                                    <dt>event:</dt><dd data-bind="text: source.eventName"></dd>
```

Comment:
I'm guessing this was removed because the event name was only exposed while we tweaked the report job, and now that it's working users should never need to change that.

Can you confirm this is the case?

Reply:
This was actually removed altogether as a feature in a previous commit and the user interface code was leftover. 

Background: We initially thought that we could build a generic interactive web page source that would monitor for a javascript event to mark completion and that would trigger render. However, it became apparent that Grafana required many more customizations to render nicely. Therefore, we nixed the plans for a general interactive web source and just created a static web source and a specialization of that (in many ways) for Grafana. We can still add a general interactive web source in the future, but there's no motivating use case for it at the moment.